forge-gui/res/cardsfolder/upcoming/city_of_the_daleks.txt

```diff
@@ -0,0 +1,15 @@
+Name:City of the Daleks
+ManaCost:no cost
+Types:Plane Skaro
+T:Mode$ AttackersDeclared | Execute$ TrigLoseLife | TriggerZones$ Command | AttackingPlayer$ You | TriggerDescription$ Whenever you attack, target opponent loses X life, where X is the number of artifacts you control.
+SVar:TrigLoseLife:DB$ LoseLife | ValidTgts$ Player | TgtPrompt$ Select target player | LifeAmount$ X
+SVar:X:Count$Valid Artifact.YouCtrl
+T:Mode$ ChaosEnsues | TriggerZones$ Command | Execute$ TrigRepeat | TriggerDescription$ Whenever chaos ensues, for each opponent, you create a 3/3 black Dalek artifact creature token with menace that attacks that opponent this turn if able. Those tokens gain haste. Sacrifice them at the beginning of the next end step.
+SVar:TrigRepeat:DB$ RepeatEach | RepeatPlayers$ Opponent | ChangeZoneTable$ True | RepeatSubAbility$ DBToken
+SVar:DBToken:DB$ Token | TokenAmount$ 1 | TokenScript$ b_3_3_a_dalek_menace | RememberTokens$ True | AtEOT$ Sacrifice | TokenOwner$ You | SubAbility$ DBPumpCombat
```

Comment:
this one might be a bit problematic

should be a single delayed trigger at least